src/libraries/Directory.Build.targets

```diff
@@ -219,10 +219,6 @@
   <Target Name="GetDocumentationFile"
           Returns="$(DocumentationFile)"/>
 
-  <PropertyGroup>
-    <DefineConstants Condition="'$(IsPrerelease)' != 'false' and '$(TargetFramework)' != '$(NetCoreAppCurrent)'">$(DefineConstants),USE_INTERNAL_ACCESSIBILITY</DefineConstants>
```

Comment:
This will probably conflict with my changes, since they currently depend on this wizardly. To avoid last-minute rebases, do you think you could revert this change and follow-up with a PR after the preview 8 snap?

Reply:
Sure :)